flink-tests/src/test/scala/org/apache/flink/api/scala/functions/ClosureCleanerITCase.scala

```diff
@@ -1,262 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.apache.flink.api.scala.functions
-
-import org.apache.flink.api.common.InvalidProgramException
-import org.apache.flink.api.scala._
-import org.apache.flink.api.scala.ExecutionEnvironment
-import org.apache.flink.core.fs.FileSystem.WriteMode
-import org.apache.flink.test.util.{MultipleProgramsTestBaseJUnit4, TestBaseUtils}
-
-import MultipleProgramsTestBaseJUnit4.TestExecutionMode
-import org.junit.{After, Before, Rule, Test}
-import org.junit.Assert.fail
-import org.junit.rules.TemporaryFolder
-import org.junit.runner.RunWith
-import org.junit.runners.Parameterized
-
-/* The test cases are originally from the Apache Spark project. Like the ClosureCleaner itself. */
-
-@RunWith(classOf[Parameterized])
-class ClosureCleanerITCase(mode: TestExecutionMode) extends MultipleProgramsTestBaseJUnit4(mode) {
```

Comment:
Can not remove this.

Reply:
I have rewritten it in Java. You can find it in `org.apache.flink.api.functions.ClosureCleanerITCase`.